src/filing/actions/fetchNewFiling.js

```diff
@@ -13,6 +14,21 @@ export default function fetchNewFiling(filing) {
       .then(json => {
         return hasHttpError(json).then(hasError => {
           if (hasError) {
+            const ignoreMessage =
+              /^The provided year or quarter is no longer available|^Bad/
```

Comment:
might want to make this configurable; backend changes that shows this message might not be aware of it being used in the frontend; further down the line we'll need to think of a way to synchronize system messages.